Wright_Will/Assignments/THEWALL/server.py

```diff
@@ -65,7 +65,7 @@ def login():
     if not bcrypt.check_password_hash(user['password'], password):
         flash("wrong password")
         return redirect('/')
-
+    session['fname'] = user['first_name']
     session['_id'] = user['id']
```

Comment:
Instead of caching the individual pieces of the user in separate vars, just cache the entire user object, eg:

session['user'] = user

Now you can get any field you want, session['user']['id'], session['user']['first_name'], etc.